### PR TITLE
Microverse > Project BookStore > Milestone 3 > FB Interactive Form

### PR DESCRIPTION
**Pull request by jurgen1c**
## In this branch:

- Add handle submit 
- Add handle change
- Add handle remove book.

### CODE REVIEW

File: src/components/BooksForm.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const BooksForm = () => {
   };
```
Comment (uche-inyama) [inline src/components/BooksForm.js:28]: You can refactor handleChange to look like so:

``` 
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}
  ```

File: src/components/BooksForm.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const BooksForm = () => {
   const [title, setTitle] = useState('');
   const [category, setCategory] = useState('');
   const dispatch = useDispatch();
-  const handleForm = e => {
+  const handleSubmit = e => {
```
Comment (uche-inyama) [inline src/components/BooksForm.js:11]: For handleSubmit function, the title field should be cleared and category set to default value, after form submission. So, include this in the function:
 
```
this.setState({
    title: '',
    category: 'Action'
  });
```

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi team :wave: 

Your project is coming along just fine. Kindly make the changes I have left in the inline comments below.

[**NB**] I was having challenges selecting the lines of code for inline comment, that's why the selection does not span the whole function.

If you have any questions, kindly reach out on slack @ Inyama, Uchechukwu Henry. 

That's it, once you're done submit another form for a code review.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

File: src/components/BooksForm.js (hunk 1 above)
Comment (jurgen1c) [reply src/components/BooksForm.js:28]: Destructed event

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations team :tada: 

**Thank you for implementing the changes required by the previous reviewer :handshake:**

**This project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Do not forget to publish your recent changes to the production server (Heroku).

Again, congratulations, you can now merge to the **development** branch.
Do not forget to fill the project completion form.

**Good luck with your next task.**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**